packages/votingV2/scripts/copy-contracts-abis.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This is awesome! Should we include:
1. An execution path within the package.json to run this. Perhaps this is combined with one of the other scripts such as first building the artefacts before deploying?
2. Documentation on when to execute this?

Reply:
This is already included in the package.json, [see this](https://github.com/UMAprotocol/subgraphs/blob/f8bec7c4eb8053d03a0557d77af66605026baa82/packages/votingV2/package.json#L10)

An also automatically run when running yarn `prepare:goerli `